### PR TITLE
[TH2-2090] Add ability to extract logs as CSV like data

### PR DESCRIPTION
**Pull request by OptimumOpium**

### CODE REVIEW

File: src/main/java/com/exactpro/th2/readlog/RegexLogParser.java [hunk 1]
```diff
@@ -15,20 +15,37 @@
  */
 package com.exactpro.th2.readlog;
 
+import java.io.StringWriter;
 import java.time.DateTimeException;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Comparator;
```
Comment (Topru333) [inline src/main/java/com/exactpro/th2/readlog/RegexLogParser.java:23]: import java.util.Comparator;
import javax.annotation.Nullable;
import org.apache.commons.text.lookup.StringLookupFactory;

Can be removed

File: src/main/java/com/exactpro/th2/readlog/Main.java [hunk 2]
```diff
@@ -66,9 +66,6 @@
 public class Main {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(Main.class);
-    private static final ObjectMapper MAPPER = new ObjectMapper()
```
Comment (Topru333) [inline src/main/java/com/exactpro/th2/readlog/Main.java:69]: Imports of ObjectMapper and all related can be removed

File: src/main/java/com/exactpro/th2/readlog/LogData.java [hunk 3]
```diff
@@ -17,6 +17,7 @@
 
 import java.time.LocalDateTime;
 import java.util.ArrayList;
+import java.util.Collection;
```
Comment (Topru333) [inline src/main/java/com/exactpro/th2/readlog/LogData.java:20]: Can be removed

**Review by Topru333 — APPROVED** [review]

**Review by Topru333 — APPROVED** [review]

File: src/main/java/com/exactpro/th2/readlog/cfg/AliasConfiguration.java [hunk 4]
```diff
@@ -101,4 +107,31 @@ public Pattern getTimestampRegexp() {
...
+        }
+        this.groupsJoinDelimiter = groupsJoinDelimiter;
+    }
+
+    public Map<String, String> getHeadersFormat() {
+        return headersFormat;
+    }
+
+    public void setHeadersFormat(Map<String, String> headersFormat) {
+        this.headersFormat = headersFormat;
```
Comment (Nikita-Smirnov-Exactpro) [inline src/main/java/com/exactpro/th2/readlog/cfg/AliasConfiguration.java:135]: Throw exception if it is empty

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Pull request merged**